signer/src/bitcoin/packaging.rs

```diff
@@ -603,6 +596,28 @@ mod tests {
             Self { votes, ..Default::default() }
         }
 
+        /// Create a new request item with random votes against.
+        fn with_rng(rng: &mut impl Rng) -> Self {
+            let mut votes = [false; 5];
+            for vote in &mut votes {
+                *vote = rng.gen_bool(0.5);
+            }
+            let needs_signature = rng.gen_bool(0.5);
+            let vsize = rng.gen_range(1..=10000);
+            let withdrawal_id = if rng.gen_bool(0.5) {
```

Comment:
nit: should we avoid generating a `withdrawal_id` if `needs_signature` is true? It shouldn't be possible to have both

Reply:
[42e7003](https://github.com/stacks-sbtc/sbtc/pull/1664/commits/42e70034d9a2d0f8f8ab861960cef2981a9b9bf4)